In [6]:
import pandas as pd 
import numpy as np 
import matplotlib.pyplot as plt 
%matplotlib inline
from IPython.display import IFrame

## Efeito Estufa

### O **efeito estufa** é o processo pelo qual a radiação da atmosfera de um planeta aquece a superfície do planeta a uma temperatura acima do que seria sem esta atmosfera.

<img src="images/CandidSelfreliantAvians.webp" width="750" align="center">

Atividades humanas como a queima de combustíveis fósseis, o emprego de certos fertilizantes, o desmatamento e o grande desperdício contemporâneo de alimentos, que têm entre seus resultados a elevação nos níveis atmosféricos de gases estufa, vêm intensificando de maneira importante o efeito estufa e desestabilizando o equilíbrio energético no planeta, produzindo um fenômeno conhecido como aquecimento global.

<img src="images/DependableCheeryBlacklemur.webp" width="750" align="center">

Aos longo dos últimos 20 anos, os satélites da Nasa monitoraram do espaço esse imenso “ser vivo”, observando não apenas suas propriedades físicas, mas o pulso da vida em diferentes estações — as ondas de calor que se movem ao redor do planeta, o gelo marinho que se expande e recua e a vegetação que floresce e padece nos continentes. 

Nas regiões árticas podem ser notadas os efeitos do aquecimento global, onde o clima está aquecendo mais rápido, os impactos já são visíveis do espaço. A vegetação de tundra do oeste do Alasca, Quebec e outros lugares ao norte está ficando mais verde, enquanto grandes incêndios entre 2004 e 2015 eliminaram milhões de hectares de florestas no Alasca. 

[Link da matéria](https://super.abril.com.br/ciencia/20-anos-de-mudancas-na-terra-em-um-gif-fascinante-da-nasa/)


<img src="images/mollweide_cycle.gif" width="750" align="center">

Atividades que geram uma determinada quantidade específica de cada gás

<img src='images/C0_1-Details-1-Emissions_By_Greenhouse_Gas.png' width="750" align="center">

### Mas afinal qual a relação das maiores economias do mundo junto a essa emissão de gases poluentes, e como fica o Brasil nessa história durante os últimos anos, pois bem, e o que vamos analisar no decorrer desta abordagem, venha comigo nesta jornada 🤗🤗🤗

<img src='images/emissao-gas-carbonico.jpg' width="750" align="center">

Vamos nessa? Let's Go Baby 😎

In [17]:
data = pd.read_csv('archive/emission_data.csv')

In [15]:
data.head()

Country  1751  1752  1753  1754  1755  1756  1757  1758  1759  \
0       Afghanistan     0     0     0     0     0     0     0     0     0   
1            Africa     0     0     0     0     0     0     0     0     0   
2           Albania     0     0     0     0     0     0     0     0     0   
3           Algeria     0     0     0     0     0     0     0     0     0   
4  Americas (other)     0     0     0     0     0     0     0     0     0   

   ...          2008          2009          2010          2011          2012  \
0  ...  8.515264e+07  9.191295e+07  1.003652e+08  1.125912e+08  1.233332e+08   
1  ...  3.183077e+10  3.301904e+10  3.421283e+10  3.541120e+10  3.664504e+10   
2  ...  2.287948e+08  2.331696e+08  2.377643e+08  2.430001e+08  2.479062e+08   
3  ...  2.894820e+09  3.015005e+09  3.132819e+09  3.252626e+09  3.380736e+09   
4  ...  7.746025e+10  7.961787e+10  8.187178e+10  8.416656e+10  8.654197e+10   

           2013          2014          2015          2016          2017  
0  1.333337e+08  1.431228e+08  1.532303e+08  1.654882e+08  1.785029e+08  
1  3.789569e+10  3.918617e+10  4.047518e+10  4.178583e+10  4.311757e+10  
2  2.529662e+08  2.586784e+08  2.646261e+08  2.708990e+08  2.772782e+08  
3  3.513171e+09  3.656348e+09  3.806940e+09  3.957319e+09  4.107870e+09  
4  8.894874e+10  9.139192e+10  9.382747e+10  9.624253e+10  9.864116e+10  

[5 rows x 268 columns]

In [22]:
print('Quantidade de linhas', data.shape[0])
print('Quantidade de colunas', data.shape[1])

Quantidade de linhas 231
Quantidade de colunas 268


In [16]:
data.columns

Index(['Country', '1751', '1752', '1753', '1754', '1755', '1756', '1757',
       '1758', '1759',
       ...
       '2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015', '2016',
       '2017'],
      dtype='object', length=268)

Podemos observar que temos dados de 268 países, desde 1751, porem nessa análise vamos olhar para os últimos 20 anos, que e onde se concentra a maior quantidade de emissão